include/boost/geometry/io/wkt/read.hpp

```diff
@@ -306,6 +306,17 @@ struct stateful_range_appender<Geometry, open>
     }
 
 private:
+    static inline bool disjoint(point_type const& p1, point_type const& p2)
+    {
+        // TODO: pass strategy
```

Comment:
OK for me, PR is large enough

Reply:
Yes, I wanted to postpone it because I didn't want to force the user to pass within point strategy into `read_wkt()`. We should probably have separate wkt strategy for that.